src/Microsoft.ML.Core/Utilities/Utils.cs

```diff
@@ -1098,5 +1105,34 @@ public static string GetDescription(this Enum value)
             }
             return null;
         }
+
+        public static int Count<TSource>(this ReadOnlySpan<TSource> source, Func<TSource, bool> predicate)
+        {
+            Contracts.CheckValue(predicate, nameof(predicate));
+
+            int result = 0;
+            for (int i = 0; i < source.Length; i++)
+            {
+                if (predicate(source[i]))
+                {
```

Comment:
>{ [](start = 16, length = 1)

remove curlies for 1-line clauses like this and below

Reply:
done.

---
In reply to: [233650638](https://github.com/dotnet/machinelearning/pull/1580#discussion_r233650638) [](ancestors = 233650638)